Library/Formula/libsearpc.rb

```diff
@@ -0,0 +1,40 @@
+class Libsearpc < Formula
+  homepage "http://www.seafile.com/"
+  url "https://github.com/haiwen/libsearpc/archive/v3.0-latest.tar.gz"
+  sha1 "cdff106886441205f46a592100ae324314127107"
+  revision 1
+
+  head "https://github.com/haiwen/libsearpc.git"
+
+  #FIX for homebrew autotools path
+  patch :DATA
+
+  depends_on "autoconf" => :build
+  depends_on "automake" => :build
+  depends_on "pkg-config" => :build
+  depends_on "libtool" => :build
+  depends_on "jansson"
+  depends_on "glib"
+
+  def install
+    system "./autogen.sh"
+    system "./configure", "--prefix=#{prefix}", "--disable-compile-demo"
+    system "make"
+    system "make", "install"
+  end
+end
+
```

Comment:
Can a test be added to do something more substantial than e.g. `--version` or `--help`? See `cmake.rb` for an example of an application formula with a good test and `tinyxml2.rb` for an example of a library formula with a good test. Thanks!